leaflet-markercluster/index.d.ts

```diff
@@ -82,7 +82,7 @@ declare namespace L {
       /*
       * Function used to create the cluster icon
       */
-      iconCreateFunction?: any;
+      iconCreateFunction?: ((cluster: L.MarkerClusterGroup) => L.Icon);
```

Comment:
This inner method should take an `L.MarkerCluster`, which should be an interface(?) that inherits from L.Marker and adds a few methods:
`getAllChildMarkers(): Marker[]`, `getChildCount(): number`, `zoomToBounds(): void`, `getBounds(): L.LatLngBounds`

ref https://github.com/Leaflet/Leaflet.markercluster/blob/master/src/MarkerCluster.js